client/src/common/date.ts

```diff
@@ -3,19 +3,20 @@
 import { DateTime } from 'luxon';
 import { ComposerTranslation } from 'vue-i18n';
 
-export function formatTimeSince(date: DateTime | undefined, t: ComposerTranslation, d: any, defaultValue='', format='long'): string {
+export function formatTimeSince(date: DateTime | undefined, t: ComposerTranslation, d: any, defaultValue='', format:'long'|'short' = 'long',
+  roundDays = false): string {
   if (!date) {
     return defaultValue;
   }
   // Get the difference in ms
   const diff = DateTime.now().diff(date, ['years', 'months', 'days', 'hours', 'minutes', 'seconds']).toObject();
 
   // More than 6 days, just display the date as is
-  if (!diff || diff.years && diff.years > 0 || diff.months && diff.months > 0 || diff.days && diff.days > 6) {
+  if (!diff || diff.years && diff.years > 0 || diff.months && diff.months > 0 || diff.days && diff.days > 6 && !roundDays) {
```

Comment:
Wouldn't
```typescript
if (!diff || diff.years && diff.years > 0 || diff.months && diff.months > 0 || diff.days && diff.days > 6) {
  return d(date.toJSDate(), format);
} else if (diff.days && roundDays) {
  return t('common.date.lastLoginDays', {days: diff.days}, diff.days);
} else if (diff.days && diff.days > 0) {
  return t('common.date.lastLoginDays', {days: diff.days}, diff.days);
}
```
work and be easier to read?

Reading this I also realized that we need some tests for this function.

Reply:
You mean added this function to the existing `if`? Because we still need `seconds`, `minutes` and `hours`.